vyked/decorators/http.py

```diff
@@ -37,7 +37,10 @@ def f(self, *args, **kwargs):
                 Stats.http_stats['total_requests'] += 1
                 if required_params is not None:
                     req = args[0]
-                    query_params = req.GET
+                    if req.method in ["POST", "DELETE", "PUT"]:
+                        query_params = yield from req.post()
```

Comment:
Most of 1mg APIs have content-type: application/json, for which req.post() will just return an empty dict, might be a better idea to req.json(), or avoid the whole functionality as query_parameters are not traditionally defined for non-GET requests, as the data is primarily sent in the request body, and need not be a dictionary.

Reply:
@amanwriter Done "sir"